### DISCUSSION TITLE
prefix singularity execution with mpirun

### DISCUSSION THREAD

**Discussion by tbugfinder** [nextflow-io/nextflow#3064] [Q&A]
Hello,

in some cases in might be beneficial to execute a singularity container within a mpirun command.
I couldn't find the option to do with with the current implementation.

Any hint would be appreciated.

Thanks & Best

Josef

**Comment by tbugfinder** [comment]
After some research I'd think this isn't possible or implemented and might require a minor code change.

**Reply by lannandc** [reply]
Was there ever an update on executing a singularity container with mpirun ?
